konig-spreadsheet/src/main/java/io/konig/spreadsheet/WorkbookLoader.java

```diff
@@ -1243,7 +1249,9 @@ private void loadPropertyConstraintRow(Row row) throws SpreadsheetException {
 			if (partitionOf != null) {
 				pathHandlers.add(new PathHandler(constraint, Konig.partitionOf, partitionOf));
 			}
-
+			if(termStatus != null) {
```

Comment:
You don't need to test whether termStatus is null.  The null check happens inside the `edge` method.